HuntingDog2016/stylesheet.css

```diff
@@ -0,0 +1,129 @@
+body {
```

Comment:
have no idea... we will see - when I can build that locally , create installer and verify then I can play with removing/adding files..